utils/token.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"io/ioutil"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+// GenerateAstarteJWTFromKeyFile generates an Astarte Token for a specific API out of a Private Key File
+func GenerateAstarteJWTFromKeyFile(privateKeyFile string, astarteService AstarteService,
+	authorizationClaims []string, expiry int64) (jwtString string, err error) {
+	keyPEM, err := ioutil.ReadFile(privateKeyFile)
+	if err != nil {
+		return "", err
+	}
+
+	return GenerateAstarteJWTFromPEMKey(keyPEM, astarteService, authorizationClaims, expiry)
+}
+
+// GenerateAstarteJWTFromPEMKey generates an Astarte Token for a specific API out of a Private Key PEM bytearray
+func GenerateAstarteJWTFromPEMKey(privateKeyPEM []byte, astarteService AstarteService,
+	authorizationClaims []string, expiry int64) (jwtString string, err error) {
```

Comment:
Maybe rename `expiry` to `ttlSeconds` or something similar so it's clear that it's not an absolute value but a relative one